src/components/learner-credit-management/BudgetDetailPageOverviewAvailability.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Button, Col, Hyperlink, ProgressBar, Row, Stack} from '@edx/paragon';
+import { formatPrice } from './data';
+import { Add, OpenInNew } from '@edx/paragon/icons';
+import {generatePath, Link} from "react-router-dom";
+import {configuration} from "../../config";
+
+const BudgetInfo = ({ label, value }) => (
+  <span className={`text-dark-500 ${label === 'Utilized' ? 'mt-auto mb-1 small' : 'display-1'}`}>
+    {value} {label === 'Utilized' ? ` ${label}` : ''}
+  </span>
+);
+
+BudgetInfo.propTypes = {
+  label: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired,
+};
+
+const BudgetDetail = ({ available, utilized, limit }) => (
+  <Stack className="border border-light-400 p-4">
+    <h4 className="text-primary-500">Available</h4>
+    <Stack direction="horizontal" gap={4} className="mt-1">
+      <span className='display-1 text-dark-700'>{formatPrice(available)}</span>
+      <span className='mt-auto mb-1 small font-family-roboto'>Utilized {formatPrice(utilized)}</span>
+    </Stack>
+    <Stack gap={2} className="mt-3">
+      <ProgressBar now={ (available - utilized) / available * 100 } variant="info" />
+      <span className="ml-auto small font-family-roboto">{formatPrice(limit)} limit</span>
+    </Stack>
+  </Stack>
+);
+
+BudgetDetail.propTypes = {
+  available: PropTypes.number.isRequired,
+  utilized: PropTypes.number.isRequired,
+  limit: PropTypes.number.isRequired,
+};
+
+const BudgetActions = ({isAssignable}) => {
+  const destinationUrl = configuration.ENTERPRISE_SUPPORT_URL;
+  
+  if (!isAssignable) {
+    return (
+      <div className="p-4">
+        <h4 className='text-primary-500'>Get people learning using this budget</h4>
+        <p>
+          Funds from this budget are set auto allocate to registered
+          learners based on settings configured with your support team.
+        </p>
+        <Button variant="outline-dark" className="mr-2 mt-3" as={Hyperlink} destination={destinationUrl} target="_blank">
+          Contact Support
+        </Button>
+      </div>
+    )
+  }
+  
+  return (
+    <div className="p-4">
+      <h4 className='text-primary-500'>Get people learning using this budget</h4>
+      <Button variant="brand" className="mr-2 mt-3" iconBefore={Add} as={Link} to={generatePath()}>
```

Comment:
The `generatePath` function is not given any arguments; the CTA button currently brings the user to the root of the MFE (http://localhost:1991) instead of to the "Catalog" tab.